src/mmw/js/src/core/setup.js

```diff
@@ -7,6 +7,9 @@
 var $ = require('jquery');
 window.jQuery = window.$ = $;
 
+require('../../shim/backbone');
```

Comment:
The reason is subtle, but these dependencies have to be included in this order. It used to work fine, until I removed some "unused" require statements caught by jshint.

For example, `router.js` references `Backbone.Marionette` but it only requires `Backbone` (not `Marionette`). 

The problem is that `Marionette` attaches itself to `Backbone`. This means that `Backbone` always has to be required before `Marionette`. Instead of doing this in every module, I decided to explicitly include them in the correct order here.